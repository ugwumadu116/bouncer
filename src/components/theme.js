import * as colors from "./color/color";
import * as fontsize from "./text/fontsize";
import * as fontweight from "./text/fontweight";
import * as lineheight from "./text/lineheight";
import * as letterspacing from "./text/letterspacing";
import * as btnSizes from "./button/button";
import * as btnState from "./button/btnState";
import * as inputsizes from './form/input/inputsizes'
import * as inputbuttonSize from './form/inputButton/inputbuttonsizes'
import * as selectSize from './form/select/selectSizes'


const theme = {
  colors,
  fontsize,
  fontweight,
  lineheight,
  letterspacing,
  btnSizes,
  btnState,
  inputsizes,
  inputbuttonSize,
  selectSize
};
export default theme;
