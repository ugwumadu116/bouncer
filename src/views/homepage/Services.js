import React from "react";
import Refund from "../../assets/refund.png";
import Shipping from "../../assets/shipping.png";
import Support from "../../assets/support.png";
import Styled from "styled-components";
import { TypographyText } from "../../components/mockups";

const Services = () => {
  const ServicesSection = Styled.section`
        max-width: 1139px;
        margin: 0px auto;
    `;
  const ServiceDivContainer = Styled.div`
        display:flex;
        margin-top:98px;
        justify-content: space-between;
        align-items: center;
    `;
  const ServiceCard = Styled.div`
        display:flex;
        height: 237px;
        flex-direction:column;
        justify-content: space-between;
        align-items: center;
    `;
  return (
    <div>
      <ServicesSection>
        <ServiceDivContainer>
          <ServiceCard>
            <img src={Refund} className="cc" alt="refund" />
            <div className="cc">
              <TypographyText
                line_height="1.8"
                color="#22262a"
                letter_spacing="normal"
                font_size="24px"
                font_weight="600"
              >
                SUPPORT 24/7
              </TypographyText>
            </div>
            <div className="cc">
              <TypographyText
                line_height="1.71"
                color="#22262a"
                letter_spacing="normal"
                font_size="14px"
                font_weight="normal"
                position="center"
              >
                Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
                eiusmod tempor minim veniam, quis nostrud reprehenderit in
                voluptate velit esse cillum dolore eu fugiat nulla pariatur
              </TypographyText>
            </div>
          </ServiceCard>
          <ServiceCard>
            <img src={Shipping} className="cc" alt="refund" />
            <div className="cc">
              <TypographyText
                line_height="1.8"
                color="#22262a"
                letter_spacing="normal"
                font_size="24px"
                font_weight="600"
              >
                SUPPORT 24/7
              </TypographyText>
            </div>
            <div className="cc">
              <TypographyText
                line_height="1.71"
                color="#22262a"
                letter_spacing="normal"
                font_size="14px"
                font_weight="normal"
                position="center"
              >
                Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
                eiusmod tempor minim veniam, quis nostrud reprehenderit in
                voluptate velit esse cillum dolore eu fugiat nulla pariatur
              </TypographyText>
            </div>
          </ServiceCard>
          <ServiceCard>
            <img src={Support} className="cc" alt="refund" />
            <div className="cc">
              <TypographyText
                line_height="1.8"
                color="#22262a"
                letter_spacing="normal"
                font_size="24px"
                font_weight="600"
              >
                SUPPORT 24/7
              </TypographyText>
            </div>
            <div className="cc">
              <TypographyText
                line_height="1.71"
                color="#22262a"
                letter_spacing="normal"
                font_size="14px"
                font_weight="normal"
                position="center"
              >
                Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
                eiusmod tempor minim veniam, quis nostrud reprehenderit in
                voluptate velit esse cillum dolore eu fugiat nulla pariatur
              </TypographyText>
            </div>
          </ServiceCard>
        </ServiceDivContainer>
      </ServicesSection>
    </div>
  );
};

export default Services;
