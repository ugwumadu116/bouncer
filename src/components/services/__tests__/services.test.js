import React from 'react'
import renderer from 'react-test-renderer'
import {ThemeProvider} from 'styled-components'
import Services from '../index'
import theme from '../../../components/theme'



it('renders correctly', ()=>{
    const tree = renderer.create(
    <ThemeProvider theme={theme}>
      <Services/>  
    </ThemeProvider>
    
    
    ).toJSON();
    expect(tree).toMatchSnapshot();
})