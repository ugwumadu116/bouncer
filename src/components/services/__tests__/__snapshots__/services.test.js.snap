// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="vis-hidden"
>
  <div
    className="sc-bdVaJa lauwRn"
  >
    <div
      className="d-flex"
    >
      <div
        className="d-flex-space-between flex-column p-3"
      >
        <div>
          <img
            alt="support"
            src="shipping.png"
          />
        </div>
        <div>
          <p
            className="sc-bwzfXH hdeUKD"
          >
            FREE SHIPPING
          </p>
        </div>
        <div>
          <p
            className="sc-bwzfXH fGlIYz text-align"
          >
            Lorem Ipsum is simply dummy text of the printing and typesetting industry.
       Lorem Ipsum has been the industry's standard dummy text ever since the 1500s,
        when an unknown printer took a galley of type and scrambled it to make a type specimen book
          </p>
        </div>
      </div>
      <div
        className="d-flex-space-between flex-column p-3"
      >
        <div>
          <img
            alt="support"
            src="refund.png"
          />
        </div>
        <div>
          <p
            className="sc-bwzfXH hdeUKD"
          >
            100% REFUND
          </p>
        </div>
        <div>
          <p
            className="sc-bwzfXH fGlIYz text-align"
          >
            Lorem Ipsum is simply dummy text of the printing and typesetting industry.
       Lorem Ipsum has been the industry's standard dummy text ever since the 1500s,
        when an unknown printer took a galley of type and scrambled it to make a type specimen book
          </p>
        </div>
      </div>
      <div
        className="d-flex-space-between flex-column p-3"
      >
        <div>
          <img
            alt="support"
            src="support.png"
          />
        </div>
        <div>
          <p
            className="sc-bwzfXH hdeUKD"
          >
            SUPPORT 24/7
          </p>
        </div>
        <div>
          <p
            className="sc-bwzfXH fGlIYz text-align"
          >
            Lorem Ipsum is simply dummy text of the printing and typesetting industry.
       Lorem Ipsum has been the industry's standard dummy text ever since the 1500s,
        when an unknown printer took a galley of type and scrambled it to make a type specimen book
          </p>
        </div>
      </div>
    </div>
  </div>
</div>
`;
